entrypoint.sh

```diff
@@ -107,14 +108,17 @@ check_cache() {
 }
 
 sync_files() {
+  # Generate the excludes list before using the output with rsync
+  local exclude_from; exclude_from="$(generate_exclude_from)"
```

Comment:
I found it preferable to generate the output in a variable here.

The text works more reliably when quoted, and while calling the function directly in the process substitution works (eg, `<(echo "generate_exclude_from)`), it produces warnings about a needless echo.